### PR TITLE
Change hyphens into em dashes for separators

### PR DESCRIPTION
**Pull request by SoNothingMC**
TL;DR: Apple uses it so why not?

# Disclaimer

I don't know JavaScript. I have built my changes and tested things out but I am not sure if I did everything correctly. Please advice if I have done any damages.

# The PR
## Right now:

Apple Music's iOS app, Web Player and iTunes all uses the U+2014 EM DASH: — symbol to separate artist with the album.
Cider is currently using a normal hyphen (U+002D HYPHEN-MINUS: -) to do this.

**Apple Music Web Player (beta)**
image

**Cider, as of commit ffc55b6b64813c58404ef762bca285b9cf567f3e**
image

## Code changes:

I have replaced most, if not all, of the hyphens separating artists and albums with em dashes. I may be missing some of them, so this PR is a draft state right now.

*Note:* Code editors and monospace fonts will shorten the em dash (longer than full-width) to full-width length, so it will look exactly the same

**Cider, after changes**
image

## Does it work?

I mean it builds and plays music fine, so ¯\\\_\(ツ\)\_\/¯

### CODE REVIEW

**Comment by cryptofyre** [comment]
Looks good. Merged.
